unittests/UnitTestFramework.xml

```diff
@@ -19,7 +19,7 @@
 ================================
 
 The framework is very simple, it consists of:
--	A set of a few (8) user classes similar to JUnit (Test, TestCase, Assert, Error, etc.)
+-	A set of a few user classes similar to JUnit (Test, TestCase, Assert, Error, etc.)

```

Comment:
no need to change now, if you end up in this file again soon, I'd be tempted to remove the word "few"